pkg/client/src/app/pages/applications/analysis-wizard/review.tsx

```diff
@@ -118,7 +132,14 @@ export const Review: React.FunctionComponent<IReview> = ({
           </DescriptionListDescription>
         </DescriptionListGroup>
         <DescriptionListGroup>
-          <DescriptionListTerm>Excluded packages</DescriptionListTerm>
+          <DescriptionListTerm>
+            {
+              // t("wizard.terms.packages")
```

Comment:
Cleanup comments.

Reply:
The comment is needed otherwise i18n removes the terms from the translation files.